tests/utils/test_logger.py

```diff
@@ -0,0 +1,57 @@
+import logging
+import os
+
+import pytest
+
+from kale.utils import logger
+
+
+@pytest.fixture(scope="session")
+def testing_logger():
+    save_dir = os.getcwd()
+    testing_logger = logger.construct_logger("test", save_dir)
+    yield testing_logger
```

Comment:
I see. So the above is run first to construct a logger. Then the test cases check the validity of the (type/existence of) resulted objects. After tests, the teardown cleans up the generated files.

Thanks for the solution. I learned a lot!